### PR TITLE
Add event description

### PR DESCRIPTION
**Pull request by alloy**
Closes https://github.com/artsy/collector-experience/issues/45

This adds a description field that accepts markdown markup _and_ will preserve line-breaks. I’ve opted to strip any other HTML, not allow images nor CSS, so that it will always have the look we prefer.

It also centers the text of both the description and the pre-existing fine-print.

screen shot 2016-12-02 at 15 08 34

### CODE REVIEW

File: app/views/rsvps/_header.html.erb [hunk 1]
```diff
@@ -4,4 +4,9 @@
   </div>
   <h5><%= @event.presented_by %></h5>
   <h1><%= @event.name %></h1>
+  <% unless @event.description.blank? %>
```
Comment (mzikherman) [inline app/views/rsvps/_header.html.erb]: #minor maybe better to have the positive check: `if @event.description.present?`

Comment (alloy) [reply app/views/rsvps/_header.html.erb]: Fair 👍

File: app/helpers/rsvps_helper.rb [hunk 2]
```diff
@@ -1,2 +1,7 @@
 module RsvpsHelper
+  def markdown(text)
+    renderer = Redcarpet::Render::HTML.new(filter_html: true, no_images: true, no_styles: true, hard_wrap: true)
+    markdown = Redcarpet::Markdown.new(renderer)
```
Comment (mzikherman) [inline app/helpers/rsvps_helper.rb:4]: #minor should these be memoized?

File: app/views/rsvps/_footer.html.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <div class="rsvp__footer">
-  <div class="rsvp__footer__fine-print">
+  <div class="rsvp__footer__fine-print center">
```
Comment (mzikherman) [inline app/views/rsvps/_footer.html.erb:2]: ©️

File: app/helpers/rsvps_helper.rb (hunk 2 above)
Comment (alloy) [reply app/helpers/rsvps_helper.rb:4]: Nah, they’re only invoked once per request, if at all.

Comment (alloy) [reply app/helpers/rsvps_helper.rb:4]: Or did you mean cached per process?

**Comment by mzikherman** [comment]
Nice! #minor comments no blockers.

File: app/helpers/rsvps_helper.rb (hunk 2 above)
Comment (alloy) [reply app/helpers/rsvps_helper.rb:4]: Did a quick benchmark, these objects are very cheap to create, so leaving it as-is.

**Comment by alloy** [comment]
Feedback addressed and also added a link to a doc with overview of Markdown syntax.

File: app/helpers/rsvps_helper.rb (hunk 2 above)
Comment (mzikherman) [reply app/helpers/rsvps_helper.rb:4]: Ah with puma yea maybe better not to.

**Comment by joeyAghion** [comment]
It would be nice if we could document the new field's expected inputs and constraints somewhere (ideally, in the code in a stable way). Questions often come up about this later and it's often difficult to comment authoritatively about the intent.

**Pull request merged**